### DISCUSSION TITLE
Let Easy mod with less than the default amount of extra lives be ranked

### DISCUSSION THREAD

**Discussion by DashDashDashDashDash** [ppy/osu#31171] [Ideas]
Currently, customizing the Easy mod to reduce the amount of extra lives (to 1 or 0) will leave a play unranked. This behavior is somewhat unexpected to me, as there is no gameplay advantage gained by reducing the mod's extra lives nor does it significantly alter the style of play as opposed to having the two extra lives.
A comparison to 1 or 0 extra lives in a regular environment would be to quit out once a life or two is depleted, so this specific mod adjustment shouldn't affect gameplay/leaderboards at all.

**Comment by MizuIro-H** [comment]
Or let it more flexible?
Lose some amount of score (like 10k) or decrease the score multiplier (like -0.02x) if a player lose a life during play with EZ enabled to punish players if they perform poorly even though it's more "forgiving"?

**Reply by fleuph** [reply]
Idk how this is contributing at all to the original suggestion. They are asking for EZ mod to still be ranked with certain mod customization options being selected which are unranked in current release. Score multipliers and punishing players for losing lives is not a part of this suggestion. It is whether or not pp and ranked score should be awarded for plays using the EZ mod with a lower number of extra lives selected.

I agree with the proposal. I don't see why this isn't already ranked.

**Reply by MizuIro-H** [reply]
I was talking about possible ways to balance it if it will be ranked.
It also means that I support this idea I think(if not I won't post this).

**Reply by fleuph** [reply]
base EZ mod is already ranked and reducing the number of extra lives doesn't change the gameplay of EZ mod. Its not like we are discussing the ranking of Wiggle, Deflate, or Depth. There's no need for balance.
